readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -58,15 +58,37 @@ using_rtd_theme = (
     ) or 'html_theme' not in globals()
 )
 if using_rtd_theme:
-    theme = importlib.import_module('{{ html_theme_import }}')
     html_theme = '{{ html_theme }}'
     html_style = None
     html_theme_options = {}
+
+# Legacy behavior for html_theme_path:
```

Comment:
This is more internal commentary that user commentary. It's probably better as a template comment block so we're not outputting debug output to users. This will be output in builds on the cat conf.py step.